### PR TITLE
Updating node.yml workflow from template

### PR DESCRIPTION
**Pull request by nextcloud-command**
Automated update of the node.yml workflow from https://github.com/nextcloud/.github

### CODE REVIEW

**Review by susnux — APPROVED** [review]

File: .github/workflows/node.yml [hunk 1]
```diff
@@ -48,17 +50,17 @@ jobs:
...
+        uses: skjnldsv/read-package-engines-version-actions@06d6baf7d8f41934ab630e97d9e6c0bc9c9ac5e4 # v3
         id: versions
         with:
           fallbackNode: '^20'
-          fallbackNpm: '^9'
+          fallbackNpm: '^10'
 
       - name: Set up node ${{ steps.versions.outputs.nodeVersion }}
-        uses: actions/setup-node@60edb5dd545a775178f52524783378180af0d1f8 # v4.0.2
+        uses: actions/setup-node@60edb5dd545a775178f52524783378180af0d1f8 # v3
```
Comment (skjnldsv) [inline .github/workflows/node.yml:63]:
```suggestion
        uses: actions/setup-node@60edb5dd545a775178f52524783378180af0d1f8 # v4.0.2
```

Comment (susnux) [reply .github/workflows/node.yml:63]: I am not sure why dependabot sometimes updates that comment and sometimes ignores it.
My guess is if it matches it updates it, if it does not it thinks it is a comment and leaves it like it was (e.g. when someone manually updated it without touching the comment).

File: .github/workflows/node.yml [hunk 2]
```diff
@@ -30,8 +32,8 @@ jobs:
           filters: |
             src:
               - '.github/workflows/**'
-              - '**/src/**'
-              - '**/appinfo/info.xml'
+              - 'src/**'
+              - 'appinfo/info.xml'
```
Comment (come-nc) [inline .github/workflows/node.yml:36]: This change should be cancelled I suppose for server?

Comment (susnux) [reply .github/workflows/node.yml]: done

**Review by come-nc — APPROVED** [review]

**Pull request closed without merging**